README.md

```diff
@@ -1,9 +1,12 @@
 # Leaderboard
 
+   This is the project build with webpack and the objective of this project is to use the APL and Use async and await JavaScript features to consume the API. As, this project is built using webpack and to open in browser you need to type `npm start` in terminal.
+
```

Comment:
- [ ] You added some description but let's try to be a little more organized. I would suggest moving this sentence:
**'As, this project is built using webpack and to open in browser you need to type `npm start` in terminal'**
to the **Getting Started** section of your README as that is where it should be

- [ ] Technically, users cannot even run `npm start` on your project without running `npm i` which is why it was suggested as a necessary change previously. Kindly add `npm i` as a necessary command under **Getting Started** section.